specs/opcode/10LT_11GT.md

```diff
@@ -0,0 +1,28 @@
+# LT & GT opcode
+## Procedure
+
+The `LT` and `GT` opcode compares the top two values on the stack, and push the
+result (0 or 1) back to the stack.
+
```

Comment:
I'd like to see a short description on

1. How do we use `c8s === b8s - a8s` to check `b8s > a8s`
2. How do we use `swap` to reuse the same gadget for GT and LT.

So that the reviewer or the auditor can have a rough idea before diving into the Python code.